code/__DEFINES/is_helpers.dm

```diff
@@ -11,6 +11,8 @@
 
 #define isweakref(D) (istype(D, /datum/weakref))
 
+#define isworld(thing) (thing == world)
```

Comment:
Why?

Reply:
I thought it was because the world could be an assoc key, but after testing with the DM bot, it doesn't work. Guess I don't need it.

Reply:
I meant why do this instead of just checking == world directly